### PR TITLE
optimized website load time

### LINKED ISSUE #1605: Optimize website load time
**Describe the bug**
The website experiences slow load times, particularly on pages with numerous images and JavaScript files. This leads to delayed content display, causing a subpar user experience.
**To Reproduce**
1. Go to the homepage or any content-heavy page .
2. Inspect load times in the browser's Developer Tools under the "Network" tab.
3. Observe the sequential loading of large images and unoptimized scripts, leading to increased load time.
4. Notice that the page takes several seconds to display content fully.

**Expected behavior**
1. Minify HTML, CSS, and JavaScript to reduce file sizes.
2. Implement Lazy Loading for images and videos to load content as needed.
3. Defer Non-Critical JavaScript using defer or async attributes.
4. Preload Critical Resources to prioritize loading key assets.

**Screenshots**

**Additional context**
The website does not currently implement browser caching, lazy loading, or script deferral, all of which could significantly reduce load time.

### PR DESCRIPTION
**Pull request by shivam8112005**
### Pull Request Checklist
Fixes #1605 

- [X] I have added screenshots and videos to show before and after the working of my code.
- [X] I have ensured that the screen size is set to 100% while making the video.
- [X] I have synced the latest fork with my local repository and resolved any conflicts.
- [X] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [X] I understand that if any the above conditions are not met , my PR will not be MERGED .

https://github.com/user-attachments/assets/7822d2c3-44e1-4cf3-8a3b-393e3b55dedc

### CODE REVIEW

**Comment by shivam8112005** [comment]
@ANSHIKA-26  is it possible to add level 2 ?

File: style.css [hunk 1]
```diff
@@ -617,15 +590,7 @@ background: var(--light);
 }
 
 
-/* Blog Card Design - Rushikesh Ghodke */
 
-/* --light: #f0e4d7;
```
Comment (ANSHIKA-26) [inline style.css:622]: add this back , it might be needed for refrence in future

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by ANSHIKA-26** [comment]
> @ANSHIKA-26 close #1378 and is it possible to add level 2 ?

sorry , cant give a level 2 for removing comments and doing some minor fixes

**Comment by shivam8112005** [comment]
ohk np wait

**Comment by shivam8112005** [comment]
is it ok now?

**Comment by shivam8112005** [comment]
@ANSHIKA-26  can you please merge it now?

**Comment by hiteashgupta1** [comment]
Refork and resync

**Comment by shivam8112005** [comment]
It means delete this repo from my acc and then refork it again?

**Pull request closed without merging**